iina/Base.lproj/Localizable.strings

```diff
@@ -143,7 +143,7 @@
 
 "alert.fatal_error" = "Fatal error: %@ \nThe application will exit now.";
 "alert.mpv_error" = "Internal error %@ (%@) when setting option %@.";
-"alert.sleep" = "Cannot prevent display sleep!";
+"alert.sleep" = "Cannot prevent display sleep!\nmacOS error 0x%0X8";
```

Comment:
I think I'd prefer a "Cannot prevent display sleep! (error code)" format, as this better matches how macOS displays errors

Reply:
I first considered doing it like that and changed to this to put emphasis on this being a macOS problem. With it now not adding any words to the string should I be updating more of the lanuages, or just base and EN? The alert now looks like:
![issue-3842-review-change](https://user-images.githubusercontent.com/86170219/186247176-6f64e3ad-42ce-4c1f-aa62-9c0c6eb0c596.jpg)

Reply:
lgtm. I prefer to not pass judgement on where the error is coming from, in general we should just present it as it is.